<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pic_Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>75</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Chụp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>75</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>281</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
